File: CS/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace WindowsApplication1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void wizardControl1_SelectedPageChanging(object sender, DevExpress.XtraWizard.WizardPageChangingEventArgs e)
        {
            if (e.Page == wizardControl1.Pages[1] && checkEdit1.Checked)
            {
                int nextPageIndex = ((e.Direction == DevExpress.XtraWizard.Direction.Forward) ? 1 : -1) + wizardControl1.Pages.IndexOf(e.Page);
                e.Page = wizardControl1.Pages[nextPageIndex];
            }
        }
    }
}

File: VB/WindowsApplication1/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub wizardControl1_SelectedPageChanging(ByVal sender As Object, ByVal e As DevExpress.XtraWizard.WizardPageChangingEventArgs) Handles wizardControl1.SelectedPageChanging
			If e.Page Is wizardControl1.Pages(1) AndAlso checkEdit1.Checked Then
				Dim nextPageIndex As Integer
				If (e.Direction = DevExpress.XtraWizard.Direction.Forward) Then
					nextPageIndex = (1) + wizardControl1.Pages.IndexOf(e.Page)
				Else
					nextPageIndex = (-1) + wizardControl1.Pages.IndexOf(e.Page)
				End If
				e.Page = wizardControl1.Pages(nextPageIndex)
			End If
		End Sub
	End Class
End Namespace